src/browser/BrowserSettingsWidget.cpp

```diff
@@ -325,16 +326,16 @@ void BrowserSettingsWidget::showProxyLocationFileDialog()
     QString fileTypeFilter(QString("%1 (*)").arg(tr("Executable Files")));
 #endif
 
-    QString sInitialFilePath = m_ui->customProxyLocation->text();
-    if (QFileInfo::exists(sInitialFilePath)) {
-        sInitialFilePath = QFileInfo(sInitialFilePath).filePath();
+    QString InitialFilePath = m_ui->customProxyLocation->text();
```

Comment:
Use `initialFilePath` as a variable name instead. No capital letters at the start.

Reply:
Done in commit 6a40a6e827266a81360c8e2860c13dffcd665a9d.